src/spec-configuration/containerFeaturesOCI.ts

```diff
@@ -7,262 +7,289 @@ import { FeatureSet } from './containerFeaturesConfiguration';
 
```

Comment:
Also converted this file to tabs.  I suggest disabling 'show whitespace changes' in the GitHub PR UI.